runtime/doc/channel.txt

```diff
@@ -1554,36 +1554,5 @@ The "params" field is optional: >
 	"params": <list|dict>
     }
 
-Depending on the use case, you can use the ch_evalexpr(), ch_sendexpr() and
```

Comment:
Why is this text (sample LSP messages) removed?

Reply:
It's a duplicate.
L1525-L1555 and L1557-L1587 are the same thing.